pkg/services/user/userimpl/store.go

```diff
@@ -218,7 +218,7 @@ func (ss *sqlStore) LoginConflict(ctx context.Context, login, email string) erro
 
 func (ss *sqlStore) loginConflict(sess *db.Session, login, email string) error {
 	users := make([]user.User, 0)
-	where := "LOWER(email)=LOWER(?) OR LOWER(login)=LOWER(?)"
+	where := "email=? OR login=?"
 	login = strings.ToLower(login)
 	email = strings.ToLower(email)
 
```

Comment:
Nit: move this to closure in `LoginConflict`. No need to have a separate function that takes a session anymore right?

Reply:
could i ask if this is for performance, readability or perhaps both.

is there a specific reason to not take in a session into a function call like the `loginConflict`